htdocs/compta/charges/index.php

```diff
@@ -271,20 +273,21 @@
 
 	print load_fiche_titre($langs->trans("VATPayments").($year ? ' ('.$langs->trans("Year").' '.$year.')' : ''), '', '');
 
-	$sql = "SELECT pv.rowid, pv.amount, pv.label, pv.datev as dm, pv.fk_bank,";
+	$sql = "SELECT ptva.rowid, pv.rowid as id_tva, pv.amount as amount_tva, ptva.amount, pv.label, pv.datev as dm, ptva.datep as date_payment, ptva.fk_bank,";
```

Comment:
It seems you replace the report that show list of vat declaration + amount paid with list of amount paid only. (vat declaration without payment disappears from report).
Is looks better to replace the title of page "Payment of vat" into "Vat declarations and payments"

Reply:
@eldy i understand, then i replaced in my last commit "INNER JOIN" by a "LEFT JOIN", so now we can see all vat declarations even if there are no payments, and now we get a similar view to social charges view.
What do you think about that ?

![image](https://user-images.githubusercontent.com/5585819/112443424-39674700-8d4d-11eb-966d-871c383fb6c3.png)